src/lib/core/ripple/ripple.ts

```diff
@@ -154,7 +154,7 @@ export class MdRipple implements OnInit, OnDestroy, OnChanges {
       // FIXME: This fails on IE11, which still sets pageX/Y and screenX/Y on keyboard clicks.
       const isKeyEvent =
           (event.screenX === 0 && event.screenY === 0 && event.pageX === 0 && event.pageY === 0);
-      this.end(event.pageX, event.pageY, isKeyEvent);
+      this.end(event.pageX, event.pageY - document.body.scrollTop, isKeyEvent);
```

Comment:
That page could be horizontally scrolled as well, so you should account for `scrollLeft` on the x-axis.

Does this still work if another element is scrolled other than `body`? Is there a unit test we can add for this? (connected position strategy has a lot of tests that deal with this type of thing).